src/current/cockroachcloud/serverless-resource-usage.md

```diff
@@ -97,26 +104,57 @@ You might also see [multiple open connections](#excessive-number-of-connections)
 
 ### Data migration
 
-Initial data ingestion during a migration may consume a high number of RUs. Generally, optimized performance will also coincide with optimized RU consumption in this case. Refer to our [Migration Overview](https://www.cockroachlabs.com/docs/v23.1/migration-overview) documentation for more information.
+Initial data ingestion during a migration may consume a high number of RUs. Generally, optimized performance will also coincide with optimized RU consumption in this case. Refer to our [Migration Overview](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/migration-overview) documentation for more information.
 
 ### Changefeeds (CDC)
 
-The {{ site.data.products.cloud }} Console does not currently provide direct observability of changefeeds, but they can contribute to significant RU usage. Refer to our documentation on [Optimizing changefeeds](https://www.cockroachlabs.com/docs/stable/cdc-queries) for performance guidance that may decrease RU consumption. CockroachDB {{ site.data.products.dedicated }} users can also [Monitor and Debug Changefeeds](https://www.cockroachlabs.com/docs/stable/monitor-and-debug-changefeeds) in the DB Console.
+Changefeeds can contribute to significant RU usage. To monitor changefeed performance, navigate to your cluster's [**Changefeeds** metrics page]({% link cockroachcloud/metrics-changefeeds.md %}) in the {{ site.data.products.cloud }} Console and monitor the various available charts.
 
-### General tips for reducing RU usage
+Refer to our documentation on [Optimizing changefeeds](https://www.cockroachlabs.com/docs/stable/cdc-queries) for performance guidance that may decrease RU consumption.
 
-The following recommendations can help reduce the RU cost of a query by reducing the work your cluster must do to execute that query:
+## General tips for reducing RU usage
+
+RU consumption can be broken down to more granular components that can give additional optimization insight. Use the {{ site.data.products.cloud }} Console [**Request Units** metrics]({% link cockroachcloud/metrics-request-units.md %}) charts to see what kind of resource usage is driving RU consumption.
+
+The following recommendations can help minimize the RU cost of a query by reducing the work your cluster must do to execute that query. The tips are grouped according to the type of resource usage.
+
+### Reads
+
+If the [**Reads** chart]({% link cockroachcloud/metrics-request-units.md %}#reads) shows high RU's for batches, requests, or bytes (payload):
 
-- Drop indexes that are no longer needed.
 - Use [secondary indexes](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/schema-design-indexes) that reduce the number of rows that need to be scanned.
-- Take advantage of SQL filters, joins, and aggregations rather than performing these operations in the application to reduce the amount of data returned to the client.
```

Comment:
I actually think this tip is more important for the Reads section than it is the Egress section (though it's valid for both sections).